verify_vk.py

```diff
@@ -30,7 +30,7 @@
 
 def getMustpassDir(api, releaseTagStr):
```

Comment:
Modifying it should be easy, just need to pass in basedir as an arg (set to "os.path.join(ctsPath, 'external', 'vulkancts', 'mustpass')" from line 62 and 67).

Then use mustpassDir = 'main' if os.path.isdir(os.path.join(basedir, 'main')) else 'master'

Reply:
Updated as suggested.